alicloud/table_alicloud_slb_load_balancer.go

```diff
@@ -0,0 +1,336 @@
+	"context"
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/errors"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/slb"
+	"github.com/sethvargo/go-retry"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAlicloudSlbLoadBalancer(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "alicloud_slb_load_balancer",
+		Description: "Alicloud Server load Balancer",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("load_balancer_id"),
+			Hydrate:    getSlbLoadBalancer,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listSlbLoadBalancers,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "load_balancer_name", Require: plugin.Optional},
+				{Name: "network_type", Require: plugin.Optional},
+				{Name: "resource_group_id", Require: plugin.Optional},
+				{Name: "master_zone_id", Require: plugin.Optional},
+				{Name: "address_ip_version", Require: plugin.Optional},
+				{Name: "v_switch_id", Require: plugin.Optional},
+				{Name: "vpc_id", Require: plugin.Optional},
+				{Name: "load_balancer_status", Require: plugin.Optional},
+				{Name: "address_type", Require: plugin.Optional},
+				{Name: "internet_charge_type", Require: plugin.Optional},
+			},
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: []*plugin.Column{
+			{
+				Name:        "load_balancer_name",
+				Type:        proto.ColumnType_STRING,
+				Description: "The name of the CLB instance.",
+			},
+			{
+				Name:        "load_balancer_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the CLB instance.",
+			},
+			{
+				Name:        "load_balancer_status",
+				Type:        proto.ColumnType_STRING,
+				Description: "The status of the CLB instance. Valid values: inactive|active|locked.",
+			},
+			{
+				Name:        "address",
+				Type:        proto.ColumnType_IPADDR,
+				Description: "The service IP address of the CLB instance.",
+			},
+			{
+				Name:        "address_type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The network type of the CLB instance. Valid values: internet|intranet.",
+			},
+			{
+				Name:        "v_switch_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the vSwitch to which the CLB instance belongs.",
+				Transform:   transform.FromField("VSwitchId"),
+			},
+			{
+				Name:        "vpc_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the virtual private cloud (VPC) to which the CLB instance belongs.",
+			},
+			{
+				Name:        "network_type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The network type of the internal-facing CLB instance. Valid values: vpc|classic.",
+				Transform:   transform.FromField("ReleaseTime").Transform(transform.UnixToTimestamp),
```

Comment:
Is the transform function used correctly here?

Reply:
Updated